### ISSUE TITLE
AZ-120_Lab01b-Azure_VM_Windows_Clustering.md - Deploy error

### ISSUE THREAD

**Issue by TesterTesterson1004** [MicrosoftLearning/AZ-120-Planning-and-Administering-Microsoft-Azure-for-SAP-Workloads#70]
# Module/Lab: 1B
#### Step: 11

Description of issue

Multiple attempts (from fresh lab start) trying to deploy the 2 DCs for this lab are failing, apparently on extension 'PepareBDC', (sic) which I have not seen before in this course.

Repro steps:

1. Take lab as written
1. Received error last night in EastUS, multiple times this morning in EastUS2 & CentralUS (does not seem to be region specific)
1. Tried workaround referenced in this steps 'note', however removing the 'PepareBDC' extension fro the BDC VM and rerunning from step 10/11 does not help. (in this case, if you attempt to redploy you have to start from the beginning, apparenlty) 
1. Have not been able to figure out anything useful whilst troubleshooting it as to why it would be failing, nor do I know how long it has been since it stopped working.

### Relevant screenshots

![image](https://github.com/MicrosoftLearning/AZ-120-Planning-and-Administering-Microsoft-Azure-for-SAP-Workloads/assets/49700217/313df83c-97ce-4e20-8486-eac1cfd7e51a)

![image](https://github.com/MicrosoftLearning/AZ-120-Planning-and-Administering-Microsoft-Azure-for-SAP-Workloads/assets/49700217/ff95b0fa-fdb6-494b-8cc7-10dd067a5ad6)

![image](https://github.com/MicrosoftLearning/AZ-120-Planning-and-Administering-Microsoft-Azure-for-SAP-Workloads/assets/49700217/3e0910ee-3578-41f7-9d00-f1d96188fd1c)

![image](https://github.com/MicrosoftLearning/AZ-120-Planning-and-Administering-Microsoft-Azure-for-SAP-Workloads/assets/49700217/dd8d4444-7aa1-4719-a9c4-7bc1ddad656a)

**Comment by SkillableKarl** [comment]
I saw same behavior, and while the workaround given in the Step 11 didn't initially work, I believe what might be happening is that the initial long (~30 minutes deploy) occurs, and if (as I did in my first test) ...the person taking the lab has become impatient and jumped ahead to start installing the next 2 VMs, which make use of the adVNET resource from the first deploy - that is probably what causes the workaround in Step 11 to fail.

In a better world, these VM deploys would *not* experience the extension install failures _at all_ and our VM deploys would 'just work' so we wouldn't have to deal with this.   Until that happens, will just have to be sure student is referred to, and has read, Step 11's 'Wait for the deployment to complete before you proceed to the next task.' note.

At least, it may be good to update the text in that step 'remove the CustomScript extension' to 'remove the PepareBDC extension', since that's the one that seems to be causing every deploy to fail:

![image](https://github.com/MicrosoftLearning/AZ-120-Planning-and-Administering-Microsoft-Azure-for-SAP-Workloads/assets/65239900/e18255c9-062a-4dad-8abf-76d68171120d)
